### DISCUSSION TITLE
scrolltop adjustment not working

### DISCUSSION THREAD

**Discussion by oeyoews** [TiddlyWiki/TiddlyWiki5#8428] [Q&A] [answered]
https://tiddlywiki.com/#Hidden%20Setting%3A%20Scroll%20Top%20Adjustment 

https://github.com/TiddlyWiki/TiddlyWiki5/blob/886514aa52ad3cdadbdec0854fad30c4d03c5e9f/core/modules/utils/dom/scroller.js#L77

* This scroll adjustment will also be affected by the animation time being zero, it will cause it to not work. @Jermolene

**Comment by Jermolene** [comment]
Hi @oeyoews could you kindly give step-by-step instructions to reproduce the problem?

**Comment by oeyoews** [comment]
![image](https://github.com/user-attachments/assets/17b09b6f-3b23-47b0-83b9-3d10b2ba4cf5)

![image](https://github.com/user-attachments/assets/c88c8566-7833-44a8-aa6f-e3c7d9ee65e9)

1. go to https://tiddlywiki.com/
2. navigate to $:/ControlPanel
![image](https://github.com/user-attachments/assets/aa36b686-bab7-41c1-83b5-142e3c4b8603)
3. modify this AnimationDuration to 0
4. and close $:/ControlPanel
5. Repeat step 2, and you will find that the scroll position becomes like this.
![image](https://github.com/user-attachments/assets/84a69be4-c373-4074-b766-38348064a868)

**Reply by Jermolene** [reply, marked as answer]
Thanks @oeyoews fixed in 685d831cb1919e9bc2169cd58a8400689d754419
